test/framefusion.test.ts

```diff
@@ -79,40 +121,64 @@ describe('framefusion', () => {
         extractor.dispose();
     });
 
-    it('should get width', async() => {
-        // Arrange
-        const extractor = await BeamcoderExtractor.create({
-            inputFileOrUrl: TEST_VIDEO,
-            outputFile: './output/frame-%04d.png',
-        });
+    it('should get source width', async () => {
+        for (let i = 0; i < ALL_TEST_VIDEOS.length; i++) {
+            // Arrange
+            const extractor = await BeamcoderExtractor.create({
+                inputFileOrUrl: ALL_TEST_VIDEOS[i],
+                outputFile: './output/frame-%04d.png'
+            });
 
-        // Act
-        const width = extractor.width;
+            // Act
+            const width = extractor.width;
 
-        // Assert
-        expect(width).to.eq(1920);
+            // Assert
+            expect(width).to.eq(ALL_TEST_VIDEO_WIDTHS[i]);
 
-        // Cleanup
-        extractor.dispose();
+            // Cleanup
+            extractor.dispose();
+        }
     });
 
-    it('Should get height', async() => {
-        // Arrange
-        const extractor = await BeamcoderExtractor.create({
-            inputFileOrUrl: TEST_VIDEO,
-            outputFile: './output/frame-%04d.png',
-        });
+    it('should get source height', async () => {
+        for (let i = 0; i < ALL_TEST_VIDEOS.length; i++) {
+            // Arrange
+            const extractor = await BeamcoderExtractor.create({
+                inputFileOrUrl: ALL_TEST_VIDEOS[i],
+                outputFile: './output/frame-%04d.png'
+            });
 
-        // Act
-        const height = extractor.height;
+            // Act
+            const height = extractor.height;
 
-        // Assert
-        expect(height).to.eq(1080);
+            // Assert
+            expect(height).to.eq(ALL_TEST_VIDEO_HEIGHTS[i]);
 
-        // Cleanup
-        extractor.dispose();
+            // Cleanup
+            extractor.dispose();
+        }
     });
 
+    it('should get frame dimensions', async() => {
```

Comment:
@antoineMoPa, so this is very surprising since they all pass. Do you know of any sources which didn't match?